### PR TITLE
Add a basic footer

### PR DESCRIPTION
**Pull request by rodaw92**

### CODE REVIEW

File: src/components/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import "./footer.css";
+import * as React from "react";
+
+export class Footer extends React.Component {
```
Comment (RedBlueKoi) [inline src/components/footer.tsx:4]: We are using Functional components

File: src/components/footer.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import "./footer.css";
+import * as React from "react";
```
Comment (RedBlueKoi) [inline src/components/footer.tsx:2]: You don't have to import React, it's not needed

File: src/components/footer.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.footer {
+    padding-top: 60px;
+}
+
+a:hover {
+    color: blue;
+    text-decoration: underline;
+}
```
Comment (RedBlueKoi) [inline src/components/footer.css:8]: Linter is still not enabled

**Comment by RedBlueKoi** [comment]
Components naming convention tells us that we should name Components with PascalCase. So in your case it's gonna be Footer.tsx. To make file more organised you can create a folder in the components section with name `footer` and put Footer.tsx and footer.css there.

File: src/components/footer.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.footer {
+    padding-top: 60px;
```
Comment (RedBlueKoi) [inline src/components/footer.css:2]: For simple styling please use Tailwind. This way all styles across the app are gonna be consistent.

File: src/components/footer.tsx (hunk 1 above)
Comment (rodaw92) [reply src/components/footer.tsx]: I have replaced it with a functional component, Footer

File: src/components/footer.css (hunk 3 above)
Comment (rodaw92) [reply src/components/footer.css]: I have used tailwind styling and deletied footer.css file

File: src/components/footer.css (hunk 4 above)
Comment (rodaw92) [reply src/components/footer.css]: Done.

File: src/components/footer.tsx (hunk 2 above)
Comment (rodaw92) [reply src/components/footer.tsx]: Deleted.

**Review by RedBlueKoi — APPROVED** [review]

**Pull request merged**